build/metadata/Dockerfile

```diff
@@ -10,7 +10,7 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM golang:1.12-alpine3.9 as builder
+FROM golang:1.13-alpine3.12 as builder
```

Comment:
Seems circleci needs to be updated as well https://app.circleci.com/pipelines/github/eclipse/che-plugin-broker/445/workflows/7469130a-3ff3-4796-baa8-a1b3981e171f/jobs/415

Reply:
Hope you don't mind that the fix is provided in https://github.com/eclipse/che-plugin-broker/pull/111/commits/f8d65b50dafc5fcd11790c827c23a68264d00fcb )

Reply:
Thanks!